### PR TITLE
Added JSON-LD in Data Catalog

### PR DESCRIPTION
**Pull request by albertoac**

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue [hunk 1]
```diff
@@ -127,6 +130,37 @@ export default {
...
     responsive () {
       return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+    },
+    jsonld () {
+      // The JSON-LD is used for SEO purposes and to index the site on Google Dataset Search
+      if (!this.dataset) {
+        return '';
+      }
+      let keyVariables = this.keyVariables && Array.isArray(this.keyVariables) ? this.keyVariables.map(elem => elem.name) : [];
+      let keywords = [...keyVariables];
```
Comment (neokore) [inline lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue:140]: Both variables could be set with `const` as you are not reassigning values

Comment (albertoac) [reply lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue]: True, I was doing re-assignments before that I ended up deleting. Thanks for noticing it!

File: lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue [hunk 2]
```diff
@@ -127,6 +130,37 @@ export default {
...
+      }
+      if (this.dataset.data_source_name) {
+        keywords.push(this.dataset.data_source_name);
+      }
+      return {
+        '@context': 'https://schema.org/',
+        '@type': 'Dataset',
+        'name': this.dataset.name,
+        'description': this.dataset.description,
+        'keywords': keywords,
```
Comment (VictorVelarde) [inline lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue:153]: I don't know about JSON-LD... does it allow a pure array for keywords? Don't you need a string with separated words ?

Comment (albertoac) [reply lib/assets/javascripts/new-dashboard/pages/Data/CatalogDataset.vue:153]: For the tests I did, it works fine. With v-html it prints all the json object correctly

**Review by VictorVelarde — APPROVED** [review]
LGTM

**Pull request merged**